samples/javascript/extensions.csproj

```diff
@@ -5,7 +5,7 @@
 	<DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="3.3.4-pre" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Kafka" Version="3.3.3" />
```

Comment:
so currently we release 3.4.0 version not 3.3.3 it was burnt and was not allowed by NuGet

Reply:
okay, then we should fix the version in all samples.

Reply:
Where headers are we need fix them, for that we can open a separate PR

Reply:
@shrohilla, I am correcting the version in this PR itself.